datasets/kor_3i4k/README.md

```diff
@@ -0,0 +1,151 @@
+task_ids:
+- intent-classification
+---
+
+# Dataset Card for 3i4K
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage: [3i4K](https://github.com/warnikchow/3i4k)**
+- **Repository: [3i4K](https://github.com/warnikchow/3i4k)**
+- **Paper: [Speech Intention Understanding in a Head-final Language: A Disambiguation Utilizing Intonation-dependency](https://arxiv.org/abs/1811.04231)**
+- **Point of Contact: [Won Ik Cho](wicho@hi.snu.ac.kr)**
+
+### Dataset Summary
+
+The 3i4K dataset is a set of frequently used Korean words (corpus provided by the Seoul National University Speech Language Processing Lab) and manually created questions/commands containing short utterances. The goal is to identify the speaker intention of a spoken utterance based on its transcript, and whether in some cases, requires using auxiliary acoustic features. The classification system decides whether the utterance is a fragment, statement, question, command, rhetorical question, rhetorical command, or an intonation-dependent utterance. This is important because in head-final languages like Korean, the level of the intonation plays a significant role in identifying the speaker's intention.
+
+### Supported Tasks and Leaderboards
+
+* `intent-classification`: The dataset can be trained with a CNN or BiLISTM-Att to identify the intent of a spoken utterance in Korean and the performance can be measured by its F1 score.
+
+### Languages
+
+The text in the dataset is in Korean and the associated is BCP-47 code is `ko-KR`.
+
+## Dataset Structure
+
+### Data Instances
+
+An example data instance contains a short utterance and it's label:
+
+```
+{
+  "label": 3,
+  "text": "선수잖아 이 케이스 저 케이스 많을 거 아냐 선배라고 뭐 하나 인생에 도움도 안주는데 내가 이렇게 진지하게 나올 때 제대로 한번 조언 좀 해줘보지"
+}
+```
+
+### Data Fields
+
+* `label`: determines the intention of the utterance and can be one of `fragment`, `statement`, `question`, `command`, `rhetorical question`, `rhetorical command` and `intonation-depedent utterance`
```

Comment:
It would be cool to make the correspondance with the label ids more eplicit
```suggestion
* `label`: determines the intention of the utterance and can be one of `fragment` (0), `statement` (1), `question` (2), `command` (3), `rhetorical question` (4), `rhetorical command` (5) and `intonation-depedent utterance` (6).
```